js/src/components/google-mc-account-card/connected-google-mc-account-card.js

```diff
@@ -85,6 +97,41 @@ const ConnectedGoogleMCAccountCard = ( {
 		removeNotice( notice.id );
 	};
 
+	const disableNotifications = async () => {
+		const { notice } = await createNotice(
+			'info',
+			__(
+				'Disabling the new Product Sync feature, please wait…',
+				'google-listings-and-ads'
+			)
+		);
+
+		try {
+			await fetchDisableNotifications();
+			invalidateResolution( 'getGoogleMCAccount', [] );
+		} catch ( error ) {
+			createNotice(
+				'error',
+				__(
+					'Unable to disconnect your Google Merchant Center account. Please try again later.',
+					'google-listings-and-ads'
+				)
+			);
+		}
+
+		removeNotice( notice.id );
+	};
+
+	const showDisconnectNotificationsButton =
+		! hideNotificationService &&
+		googleMCAccount.wpcom_rest_api_status === 'approved';
+
+	const showErrorNotificationsNotice =
+		! hideNotificationService &&
+		googleMCAccount.wpcom_rest_api_status === 'error';
```

Comment:
I was thinking whether it would be better to display an error footer by checking if `wpcom_rest_api_status` is not equal to `approved`. This could be useful as there might be other statuses like `disapproved` by the merchant.